### PR TITLE
Getting started README

### PR DESCRIPTION
**Pull request by McDonaldsHappyMeal**
Created a getting_started.md file, that goes into a little bit of detail on topics PacSim subscribes to, what they mean, and regurgitates some other information already present in the main README and the ARWO presentation slides.

If there's anything you'd like me to change, please let me know :>

### CODE REVIEW

File: doc/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Getting started with PacSim
+
+To use PacSim, you need to create a message converter node to match your own interfaces with the simulator. You will need to advertise and publish to some topics. Note that some topics are not directly used in the simulator.
+
+| TOPIC                        | MESSAGE TYPE               | DESCRIPTION                                                                           | USED?  |
+|------------------------------|----------------------------|---------------------------------------------------------------------------------------|---|
+| /pacsim/steering_setpoint    | pacsim::msg::StampedScalar | Target steering angle at the steering wheel/sensor (rad)                              | YES |
+| /pacsim/torques_max          | pacsim::msg::Wheels        | Upper bound torque value at each wheel's motor (Nm)                                   | YES |
+| /pacsim/powerground_setpoint | pacsim::msg::StampedScalar | Powered ground multiplier. 0 for no powered aero, 1 for full capacity (Dimensionless) | YES |
```
Comment (alexphieler) [inline doc/getting_started.md:11]: Please put this in the last row so that related topics are grouped together

File: doc/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The example launch file (example.launch.py) shows an example of how to start the simulator node and the robot_state_publisher for the 3D visualisation of the car.
+
+The sensors and vehicle model are configured using config files. Examples are provided in the config folder. Things such as the discipline or the path of the track file or config files are defined using ROS2 parameters.
+
+Future improvements that you may like to consider include, but are not limited to:
+- Implement a more accurate vehicle model in the future via the `IVehicleModel` class.
+- Improve sensor modelling (e.g. realistic false positive and false negative cone detections)
+- Generate ground truth reference trajectory from track YAML files
+- External clock control (ability to pause/play at will for debugging purposes)
```
Comment (alexphieler) [inline doc/getting_started.md]: Already implemented or not relevant for getting started (these are more general enchancements and not customizations)

File: doc/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+| /pacsim/wheelspeed_setpoints | pacsim::msg::Wheels        | Target wheel speeds at each wheel's motor (RPM)                                       | NO |
+
+This simulator does not model the interface of a motor controller. It is highly recommended you implement your own internal control loop that controls the amount of torque applied to each wheel to reach a given target wheel speed. Moreover, `torques_max` can be seen more as the "current torque request sent", rather than the upper bound torque value at each wheel's motor.
+
+The example launch file (example.launch.py) shows an example of how to start the simulator node and the robot_state_publisher for the 3D visualisation of the car.
+
+The sensors and vehicle model are configured using config files. Examples are provided in the config folder. Things such as the discipline or the path of the track file or config files are defined using ROS2 parameters.
+
+Future improvements that you may like to consider include, but are not limited to:
+- Implement a more accurate vehicle model in the future via the `IVehicleModel` class.
```
Comment (alexphieler) [inline doc/getting_started.md]: I prefer the old formulation

> The default vehicle model provided is rather simple and just meant to be a starting point. You are encouraged to integrate your own vehicle model by implementing the `IVehicleModel` class

File: doc/getting_started.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+| /pacsim/steering_setpoint    | pacsim::msg::StampedScalar | Target steering angle at the steering wheel/sensor (rad)                              | YES |
+| /pacsim/torques_max          | pacsim::msg::Wheels        | Upper bound torque value at each wheel's motor (Nm)                                   | YES |
+| /pacsim/powerground_setpoint | pacsim::msg::StampedScalar | Powered ground multiplier. 0 for no powered aero, 1 for full capacity (Dimensionless) | YES |
+| /pacsim/torques_min          | pacsim::msg::Wheels        | Lower bound torque value at each wheel's motor (Nm)                                   | NO |
+| /pacsim/wheelspeed_setpoints | pacsim::msg::Wheels        | Target wheel speeds at each wheel's motor (RPM)                                       | NO |
+
+This simulator does not model the interface of a motor controller. It is highly recommended you implement your own internal control loop that controls the amount of torque applied to each wheel to reach a given target wheel speed. Moreover, `torques_max` can be seen more as the "current torque request sent", rather than the upper bound torque value at each wheel's motor.
```
Comment (alexphieler) [inline doc/getting_started.md]: I would use a more generalized formulation for this, kind of like that:

The interfaces are provided such that motor controllers with an internal control loop for the wheelspeeds (e.g. AMK or Electrophorus Engineering) can be fully modelled in the simulator.

The default model doesn't contain this control loop. The values provided for `torques_max` are the actual torques applied to the wheels.

File: doc/getting_started.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Getting started with PacSim
+
+To use PacSim, you need to create a message converter node to match your own interfaces with the simulator. You will need to advertise and publish to some topics. Note that some topics are not directly used in the simulator.
```
Comment (alexphieler) [inline doc/getting_started.md]: for the last sentence:

Note that some topics are not used in the simulator with the default vehicle model provided.

File: doc/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+The sensors and vehicle model are configured using config files. Examples are provided in the config folder. Things such as the discipline or the path of the track file or config files are defined using ROS2 parameters.
+
+Future improvements that you may like to consider include, but are not limited to:
+- Implement a more accurate vehicle model in the future via the `IVehicleModel` class.
+- Improve sensor modelling (e.g. realistic false positive and false negative cone detections)
+- Generate ground truth reference trajectory from track YAML files
+- External clock control (ability to pause/play at will for debugging purposes)
+
+### Do you have any ideas? Please feel free to contribute by creating an issue on the repository!
```
Comment (alexphieler) [inline doc/getting_started.md]: I think it's redundant since it's mentioned in the main README.md

**Comment by alexphieler** [comment]
I think it makes sense to have a section for the simulator inputs and outputs each. Could you add that structure to the README and just put a "TBD" placeholder for the outputs?

- Getting started
  - Interfaces
    - Inputs
    - Outputs
  - Other notes

**Comment by alexphieler** [comment]
Also in the table, could you replace "USED" with "USED WITH DEFAULT MODEL"?

File: doc/getting_started.md (hunk 1 above)
Comment (McDonaldsHappyMeal) [reply doc/getting_started.md:11]: Sorry, I don't fully understand the order in which you want me to put these topics, but I will recommit with a newly sorted one and see if you like it. If you don't like it, let me know.

**Pull request merged**